### PR TITLE
Added Legends of Equestria Presence

### PR DESCRIPTION
**Pull request by TheSlowly1**

### CODE REVIEW

File: Legends of Equestria/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "author": {
+        "name": "TheSlowly",
+        "id": "505071508851523599"
+    },
+    "service": "LoE",
```
Comment (Bas950) [inline Legends of Equestria/dist/metadata.json]: service name needs to be the same as folder name

**Comment by Bas950** [comment]
Also please upload proof that it works.

**Comment by TheSlowly1** [comment]
> Also please upload proof that it works.

https://i.imgur.com/0nZ9Aw9.png.
It has, however, problems at first entering the website. A mistake that don't seem to be caused by the  code. After a first interaction it works, though

File: Legends of Equestria/presence.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+var presence = new Presence({
+  clientId: "651406405093425152",
+  mediaKeys: false
+}), presenceData: presenceData = {
+  largeImageKey: "logo"
+};
+
+presence.on("UpdateData", async () => {
+  console.log(document.location.pathname);
```
Comment (Bas950) [inline Legends of Equestria/presence.ts]: maybe remove console log xD

Comment (TheSlowly1) [reply Legends of Equestria/presence.ts]: kms tried to debug that error

**Comment by frozaum** [comment]
hot

**Pull request merged**